### DISCUSSION TITLE
Increase distance between camera and player

### DISCUSSION THREAD

**Discussion by devl0f** [Sixze/ALS-Refactored#234] [Q&A] [answered]
Hello! Am trying to scale the distance between the player and camera based on user viewport (so that ultrawide monitors are supported properly) but cannot find any way of reliably altering the distance between the camera and the player at runtime. Does anyone know the best practice to achieve this?

**Comment by colorindarkness** [comment, marked as answer]
You can adjust everything in the Camera Animation Blueprints. AB_Camera I believe.

**Reply by devl0f** [reply]
I did play around a bit with the CameraOffsetX curve modifier but it doesn't seem as if its respected, turns out I wasn't looking deep enough in the blueprint. The settings can be altered in the velocity direction and looking direction states. Thanks for the help! :)

**Comment by EddieSoloProjects** [comment]
I'm trying to make the camera move a little closer to the player and no matter what I change in the anim blueprint it stays in the same place.
